src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__tableau_usage.sql

```diff
@@ -0,0 +1,25 @@
+select
+    srh.employee_number,
+    srh.formatted_name,
+    srh.job_title,
+    srh.assignment_status,
+    srh.home_work_location_name as location,
+    srh.home_work_location_abbreviation as location_shortname,
+    srh.home_work_location_grade_band as grade_band,
+    srh.home_business_unit_name as entity,
+    srh.home_department_name as department,
+    srh.reports_to_formatted_name as manager,
+
+    vc.name_hist_projects as `project`,
+    vc.name_hist_workbooks as workbook,
+    vc.name_hist_views as `view`,
+    vc.`action`,
+    vc.created_at_local_new_york as viewed_at,
+    vc.id as view_count_id,
+    concat('https://tableau.kipp.org/t/KIPPNJ/views/', vc.view_url, '?:embed=y') as url,
+from {{ ref("stg_tableau__view_count_per_view") }} as vc
+left join
+    {{ ref("int_people__staff_roster_history") }} as srh
+    on lower(vc.`user_name`) = lower(srh.sam_account_name)
```

Comment:
I think `sam_account_name` is already made lowercase in `int_people__staff_roster_history`, but `lower(user_name)` should be moved to `stg_tableau__view_count_per_view`